docs/ruleset_functions/get_HVAC_systems_applicable_10cd.md

```diff
@@ -0,0 +1,82 @@
+## get_HVAC_systems_applicable_10cd
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 c and d are applicable.
+
```

Comment:
@claperle to clarify, we can add the following descriptive text tto the rule description
Where no heating system exists or no heating system has
been submitted with design documents, the system type
shall be the same system as modeled in the baseline building
design and shall comply with but not exceed the requirements
of Section 6.
d. Where no cooling system exists or no cooling system has
been submitted with design documents, the cooling system
type shall be the same as modeled in the baseline building
design and shall comply with the requirements of Section 6

Reply:
@supriyagoel I do not think this function is used anywhere. Please let me know if I am mistaken. I have not looked at it since early Feb.